graph/Graph/Graph.h

```diff
@@ -0,0 +1,35 @@
+//
+// Created by emyrmak on 7/9/2019.
+//
+#include <limits>
+#include <vector>
+
+class Graph {
+private:
+    const int infinity = std::numeric_limits<int>::max();
+    int startVertex;
+    int endVertex;
+    const int size;
+    int** matrix;
+
+    bool* visited;
+    int* distance;
```

Comment:
`matrix`, `visited` and `distance` should be `std::vector`s:
`std::vector<bool> visited`
`std::vector<int> distance`
`std::vector<std::vector<int>> matrix`